### PR TITLE
[Nataliia Falinska] JS-task

### PR DESCRIPTION
**Pull request by NFalinska**

### CODE REVIEW

File: js-practice/sass/_mediaLaptop.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
+@media only screen and (max-width: 1024px) {
+  /* Header */
+  .top-bar {
+    .flex {
```
[the remark this answers is not in the document]
Comment (NFalinska) [reply js-practice/sass/_mediaLaptop.scss]: It isn't my sass file. I added only js-file and some changes in html-file

File: js-practice/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable */
+const menu = document.querySelector('.menu-btn');
+const hiddenMenu = document.querySelector('.burger-menu');
+menu.onclick = () => {
+  hiddenMenu.classList.contains('d-none') ? hiddenMenu.classList.remove('d-none') : hiddenMenu.classList.add('d-none');
```
Comment (AlyonaPianykh) [inline js-practice/index.js]: Just use ```toggle```

File: js-practice/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  }
+};
+
+document.querySelector('.tabs-header').addEventListener('click', changeTabs);
+
+function changeTabs(event) {
+  const dataTab = event.target.getAttribute('data-tab');
+  const noActive = document.getElementsByClassName('btn-info');
+  const tabBody = document.getElementsByClassName('tab-b');
+  if (event.target.className == 'btn-info') {
```
Comment (AlyonaPianykh) [inline js-practice/index.js]: What if there would be more then 1 class? This condition is not very good

File: js-practice/index.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  contactForm[i].addEventListener("blur", validate, false);
+};
+
+function validate(event) {
+  if (!event.target.validity.valid || !event.target.value) {
+    event.target.classList.add('invalid');
+    alert('Please fill out the form');
+  } else {
+    event.target.classList.remove('invalid');
+    event.target.classList.add('valid');
```
Comment (AlyonaPianykh) [inline js-practice/index.js]: you can chain methods ```event.target.classList.remove('invalid').add('valid')```

File: js-practice/index.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+};
+const contactForm = document.getElementsByClassName("message");
+for (i = 0; i < contactForm.length; i++) {
+  contactForm[i].addEventListener("blur", validate, false);
+};
+
+function validate(event) {
+  if (!event.target.validity.valid || !event.target.value) {
+    event.target.classList.add('invalid');
+    alert('Please fill out the form');
```
Comment (AlyonaPianykh) [inline js-practice/index.js]: Never ever use alert for showing input errors!!

**Pull request merged**